lib/internal/errors.js

```diff
@@ -54,7 +61,6 @@ function message(key, args) {
 // Utility function for registering the error codes. Only used here. Exported
 // *only* to allow for testing.
 function E(sym, val) {
-  assert(messages.has(sym) === false, `Error symbol: ${sym} was already used.`);
```

Comment:
Because of issues in the loading order of the `errors.js` module relative to `assert.js` on startup. There is a circular dependency that happens that causes the `util.js` used within `assert.js` to fail depending on when `assert` is loaded. I could replace this with a simple throw rather than the call to assert if you'd be more comfortable with that.